packages/design-system/src/components/Skeleton/variations/SkeletonInput.tsx

```diff
@@ -0,0 +1,11 @@
+import SkeletonPrimitive, { SkeletonPrimitiveProps } from '../Primitive/Skeleton.Primitive';
+import styles from './SkeletonInput.module.scss';
+import React, { forwardRef, Ref } from 'react';
+
+export type SkeletonInputProps = Omit<SkeletonPrimitiveProps, 'className'>;
+
+const SkeletonInput = forwardRef((props: SkeletonInputProps, ref: Ref<HTMLSpanElement>) => {
+	return <SkeletonPrimitive isBlock className={styles.skeletonInput} {...props} ref={ref} />;
+});
```

Comment:
```suggestion
	return <StackVertical gap="XXS" as="span" {...props} ref={ref}>
	    <SkeletonHeading size="XS" />
	    <SkeletonPrimitive isBlock className={styles.skeletonInput} />
	</StackVertical>;
});
```
Or something like that